packages/@n8n/permissions/package.json

```diff
@@ -19,5 +19,8 @@
   "types": "dist/index.d.ts",
   "files": [
     "dist/**/*"
-  ]
+  ],
+  "dependencies": {
```

Comment:
what was the reason for this change?

Reply:
The change was initially automated to track the introduction of a new file. However, the file has since been removed.